index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Crystal-Collector</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <style>
+        body {
+            background-color: black;
+        }
+
+        img {
+            vertical-align: middle;
+            border-style: none;
+            height: 200px;
+
+        }
+    </style>
+</head>
+
+<body>
+
+    <div class="container" style="margin-bottom: 30px;">
+        <div class="row">
+            <h1 style="color:white;
```

Comment:
You can make a CSS file to modify the elements within that file.  The html work fine as well but it is less code if you split the files.